source/atlas-cli-changelog.txt

```diff
@@ -14,6 +14,43 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.9.1:
+
+{+atlas-cli+} 1.9.1
```

Comment:
can we add an extra note that this will be the last release available in the 4.2 linux (debian, ubuntu and rhel) repositories?